src/utils/ApiClient.js

```diff
@@ -97,6 +98,9 @@ export default class ApiClient {
                 .post(`${this.server}/rest/v4/${partialUrl}`)
                 .auth(this.username, this.password)
                 .set("User-Agent", `Hyperwallet Node SDK v${this.version}`)
+                .set("x-sdk-version", this.version)
+                .set("x-sdk-type", "NodeJS")
+                .set("x-sdk-contextId", uuidv4())
```

Comment:
the contextId UUID value should be created as part of the constructor class and any HTTP operation done in the same ApiClient instance should use the same.

Reply:
Done